tests/src/p2p/hipPeerToPeer_simple.cpp

```diff
@@ -23,10 +23,10 @@ THE SOFTWARE.
 // Also serves as a template for other tests.
 
 /* HIT_START
- * BUILD: %t %s ../test_common.cpp
- * RUN: %t EXCLUDE_HIP_PLATFORM all
- * RUN: %t --memcpyWithPeer EXCLUDE_HIP_PLATFORM all
- * RUN: %t --mirrorPeers EXCLUDE_HIP_PLATFORM all
+ * BUILD: %t %s ../test_common.cpp 
+ * RUN: %t EXCLUDE_HIP_PLATFORM hipcc 
```

Comment:
Not sure what you mean by ```EXCLUDE_HIP_PLATFORM hipcc```. Only valid values are ```hcc```, ```nvcc``` and ```all```.

Reply:
@phani544 can you update the code or respond to the above comment?

Reply:
@mangupta, I will modify the test shortly.

Reply:
@phani544 could please push the updated code?